website/project/model.py

```diff
@@ -4290,15 +4292,30 @@ def _email_template_context(self, user, node, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        log_type = NodeLog.PROJECT_REGISTERED
+        submitted_time = None
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+            is_prereg = "Prereg" in draft.registration_schema.name
```

Comment:
Since we ended up having to do more than set `log_type` just below, we can fold this down into the condition itself and not assign `is_prereg` here. Does that make sense?